Debian package 1.0.0 for jessie 64bit is broken

Man pages in `usr/share/man/man1/` inside the package are all just empty gzip files.

When I tried to rebuild the 1.1.0 Debian packages from the release tarball I got stuck with 
```
[...]
make[1]: Entering directory `/tmp/git-lfs-1.1.0'
mkdir -p debian/git-lfs/usr/bin
cp obj-x86_64-linux-gnu/bin/git-lfs debian/git-lfs/usr/bin/
make[1]: Leaving directory `/tmp/git-lfs-1.1.0'
   dh_install -O--buildsystem=golang
   dh_installdocs -O--buildsystem=golang
   dh_installchangelogs -O--buildsystem=golang
   dh_installexamples -O--buildsystem=golang
   dh_installman -O--buildsystem=golang
Converting man pages into code...
Failed to create go file: open ../commands/mancontent_gen.go: no such
file or directory
Converting man pages into code...
Failed to open man dir: <nil>
dh_installman: man --recode UTF-8 ./git\-lfs\-checkout\.1 >
git\-lfs\-checkout\.1\.new returned exit code 2
make: *** [binary] Error 2
dpkg-buildpackage: error: debian/rules binary gave error exit status 2
```
where `man` seems to be contained in the tarball.

Maybe this may be related to the man pages never showed up in the first place.

--- a/script/changelog
+++ b/script/changelog
@@ -34,16 +34,32 @@ misc=""
 for rev in $(git rev-list --merges $range); do
   git show $rev
 
-  echo ""
-  echo "Categorize this change: [f,b,m,s] ?"
+  processed=0
+  while [ $processed -eq 0 ]; do
+    echo "Categorize this change: [f,b,m,s,?] ?"
+    read -n 1 opt
+    echo ""
 
-  read -n 1 opt
+    case $opt in
+      [fbms])
+        processed=1
+        ;;
+      ?)
+        echo "f - mark this merge as a feature"
+        echo "b - mark this merge as a bugfix"
+        echo "m - make this merge as a misc. change"
+        echo "s - skip this merge, excluding it from the changelog"
+        echo "? - display this help message"
+        ;;
+      *)
+        echo "Unknown option: $opt, try again."
+        ;;
+    esac
+  done
 
-  if [ "$opt" = "s" ]; then
-    continue
+  if [ $opt != "s" ]; then
+    summary="$(commit_summary $rev)"
   fi
-
-  summary="$(commit_summary $rev)"
 
   case $opt in
     f)
@@ -55,9 +71,6 @@ for rev in $(git rev-list --merges $range); do
     m)
       misc="$(printf "%s\n%s\n" "$misc" "$summary")"
       ;;
-    *)
-      echo "Usage: [f,b,m,s]"
-      exit 1
   esac
 done
 

--- a/script/changelog
+++ b/script/changelog
@@ -11,7 +11,7 @@ commit_summary() {
   id="$(echo $prjson | jq -r -e ".number")"
   author="$(echo $prjson | jq -r -e ".user.login")"
 
-  echo "- $title #$id (@$author)"
+  echo "* $title #$id (@$author)"
 }
 
 range=$1

--- a/script/changelog
+++ b/script/changelog
@@ -11,6 +11,12 @@ commit_summary() {
   id="$(echo $prjson | jq -r -e ".number")"
   author="$(echo $prjson | jq -r -e ".user.login")"
 
+  # If the title begins with "Backport", then strip everything until the actual
+  # pull-request title.
+  if grep -q "Backport" <(echo $title); then
+    title="$(echo $title | sed 's/^[^:]*: //g')"
+  fi
+
   echo "* $title #$id (@$author)"
 }
 

--- a/script/changelog
+++ b/script/changelog
@@ -1,0 +1,66 @@
+#!/usr/bin/env bash
+#
+# Interactively generates a changelog over a range of commits:
+
+commit_summary() {
+  local hash=$1
+
+  pr=$(git show $hash | grep -o "#\([0-9]*\)" | cut -c 2-)
+  prjson="$(curl -n https://api.github.com/repos/github/git-lfs/pulls/$pr 2>/dev/null)"
+  title="$(echo $prjson | jq -r -e ".title")"
+  id="$(echo $prjson | jq -r -e ".number")"
+  author="$(echo $prjson | jq -r -e ".user.login")"
+
+  echo "- $title #$id (@$author)"
+}
+
+range=$1
+
+if [ "$range" = "" ]; then
+  echo "Usage: $0 [options] base..next"
+  exit 1
+fi
+
+features=""
+bugs=""
+misc=""
+
+for rev in $(git rev-list --merges $range); do
+  git show $rev
+
+  echo ""
+  echo "Categorize this change: [f,b,m,s] ?"
+
+  read -n 1 opt
+
+  if [ "$opt" = "s" ]; then
+    continue
+  fi
+
+  summary="$(commit_summary $rev)"
+
+  case $opt in
+    f)
+      features="$(printf "%s\n%s\n" "$features" "$summary")"
+      ;;
+    b)
+      bugs="$(printf "%s\n%s\n" "$bugs" "$summary")"
+      ;;
+    m)
+      misc="$(printf "%s\n%s\n" "$misc" "$summary")"
+      ;;
+    *)
+      echo "Usage: [f,b,m,s]"
+      exit 1
+  esac
+done
+
+echo "" >&2
+
+cat <<- EOF
+# Features$features
+
+# Bugs$bugs
+
+# Misc$misc
+EOF

--- a/script/changelog
+++ b/script/changelog
@@ -58,9 +58,12 @@ done
 echo "" >&2
 
 cat <<- EOF
-# Features$features
+### Features
+$features
 
-# Bugs$bugs
+### Bugs
+$bugs
 
-# Misc$misc
+### Misc
+$misc
 EOF

